build.ps1

```diff
@@ -178,6 +182,10 @@ function Msvs
     $VXXCommonTools = $null
     $CmakeGenerator = $null
 
+	# https://github.com/Microsoft/vswhere/commit/a8c90e3218d6c4774f196d0400a8805038aa13b1 (Release mode / VS 2015 Update 3)
```

Comment:
My target is to strip all the binaries from this repository eventually (even pruning the old ones aka #35).

I'd prefer to download `vswhere` if required. Looks like there is a official release at https://github.com/Microsoft/vswhere/releases/tag/1.0.47 (same version was pushed to `nuget.org`).

Reply:
Some code to check and download `vswhere` from a different script I wrote. Someone can adapt this if they wanted.

```
$programFilesDir = (${env:ProgramFiles(x86)}, ${env:ProgramFiles} -ne $null)[0]

$client = New-Object System.Net.WebClient;

$vswherePath = Join-Path $programFilesDir 'Microsoft Visual Studio\Installer\vswhere.exe'
#Check if we already have vswhere which is included in newer versions of VS2017 installer
if(-not (Test-Path $vswherePath))
{
    # Download vswhere if we don't have a copy
    $vswherePath = Join-Path $WorkingDir \vswhere.exe
    
    # TODO: Check hash and download if hash differs
    if(-not (Test-Path $vswherePath))
    {
	    $client.DownloadFile('https://github.com/Microsoft/vswhere/releases/download/2.2.7/vswhere.exe', $vswherePath);
    }
}
```